tools/SourceKit/lib/SwiftLang/SwiftASTManager.cpp

```diff
@@ -404,6 +405,13 @@ bool SwiftASTManager::initCompilerInvocation(CompilerInvocation &Invocation,
   ClangImporterOptions &ImporterOpts = Invocation.getClangImporterOptions();
   ImporterOpts.DetailedPreprocessingRecord = true;
 
+  llvm::InitializeAllTargets();
+  llvm::InitializeAllTargetMCs();
+  llvm::InitializeAllAsmPrinters();
+  llvm::InitializeAllAsmParsers();
```

Comment:
Oh, actually, I think I might've put this in for when I tested with in-proc SourceKit. Is there a better place I can put this that would work for both?